From d4b60804f9677782976d3cb565e3a8557776eb36 Mon Sep 17 00:00:00 2001
From: Bhushan Shah <bhush94@gmail.com>
Date: Thu, 28 Jan 2021 15:35:33 +0530
Subject: [PATCH] Fix logic error in f19f629b4ba85

(cherry picked from commit acda20ffe5ced402cce269d623b4b910cd6a6dfc)
---
 shell/contents/updates/panelsfix.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shell/contents/updates/panelsfix.js b/shell/contents/updates/panelsfix.js
index 21d1fdc..0ad9801 100644
--- a/shell/contents/updates/panelsfix.js
+++ b/shell/contents/updates/panelsfix.js
@@ -6,7 +6,7 @@ for (let i in panels()) {
     if (panels()[i].type === "org.kde.phone.panel") {
         topFound = true;
     } else if (panels()[i].type === "org.kde.phone.taskpanel") {
-        topFound = true;
+        bottomFound = true;
     }
 }
 
-- 
GitLab

